packages/admin-ui-docs/src/components/PageSection/index.tsx

```diff
@@ -29,10 +30,10 @@ export function PageSection(props: PageSectionProps) {
     >
       <div className="section-info">
         <div className="header">
-          <Heading id={props.id} as="h2">
+          <Heading id={props.id} as={`h${props.level || 2}`}>
```

Comment:
Just a suggestion to be more readable, feel free to accept or not
```suggestion
          const headingLevel = `h${props.level ?? 2}`
...
          <Heading id={props.id} as={headingLevel}>
```